queries.sql

```diff
@@ -14,4 +14,28 @@ SELECT name FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: 
>Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Verify that changes were made.
Commit the transaction.
Verify that changes persist after commit.

- [ ] You did not have the queries for the above requirements in your `queries.sql` file. I'll suggest you add the queries to the file. :smile: :small_airplane:

Reply:
thank you @V-Blaze 💯 for your valuable feedback